packages/mobile/src/services/offline-downloader/offline-downloader.ts

```diff
@@ -245,9 +245,11 @@ export const downloadTrack = async (trackForDownload: TrackForDownload) => {
 
   try {
     if (await verifyTrack(trackIdStr, false)) {
+      const trackJson = await getTrackJson(trackIdStr)
+      if (!trackJson) return
+
       // Track is already downloaded, so rewrite the json
```

Comment:
super nit - move this comment back above ?

Reply:
so i didn't because technically trackJson is nullable, in which case it's not undefined. what you think?

Reply:
ohh yea makes sense!